src/PantherTestCaseTrait.php

```diff
@@ -161,12 +167,18 @@ protected static function createPantherClient(array $options = [], array $kernel
             }
         }
 
+        // Only allow browser arguments if a browser has been explicitly specified
```

Comment:
Why would you have to explicitly set the used browser to provide browser arguments ?

Reply:
This is a question I've asked myself. I'm not sure if passing a wrong argument would cause any issues. May be this shouldn't be enforced at all.

Reply:
I think users should be able to rely on chrome being the default, so providing arguments without explicitly setting the browser should be allowed.

Reply:
Ok, I agree! I'm not expert on browser drivers, so I just wanted to get users into a hypotetical crash due to passing wrong arguments. I don't think this would ever happen, so I can remove this altogether.